CHANGELOG.md

```diff
@@ -309,6 +311,8 @@ instances according to the [policy configuration](https://github.com/nasa/cumulu
     element that has a `Type` of either `"Update"` or `"Insert"`.  If neither are
     present, the granule's last update date falls back to the `"Create"` type
     provider date, or `undefined`, if none is present.
+- CVE-2022-2477
+  - Update node-forge to 1.3.0 in `@cumulus/common` to address CVE-2022-2477
```

Comment:
We probably don't need to include this manually because it will get added when we merge main back to this branch

Reply:
I can remove this commit